hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/BaseZookeeperBasedLockProvider.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Do you want to update the docs of `DynamoDbBasedLockConfig#MAX_PARTITION_KEY_SIZE_BYTE`?

Reply:
done